test/unittests/testData/ngsi9.discoverContextAvailabilityRequest.entityIdTypeAsField.invalid.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file wasn't meant to be there, its presence here is due to a bug in the git implementation for Mac; I can't get rid of it. Please, one of you, remove it from the PR.